### PR TITLE
chore: Update remaining Typescript documentation snippets

### PR DESCRIPTION
**Pull request by mutahhir**

### CODE REVIEW

File: examples/typescript/documentation/functions.ts [hunk 1]
```diff
@@ -3,25 +3,58 @@
...
 
+    // DOCS_BLOCK_START:functions
     new TerraformOutput(this, "first-zone", {
       value: Fn.element(zones.names, 0),
     });
+    // DOCS_BLOCK_END:functions
+
+    // DOCS_BLOCK_START:operators
+
+    // ...
```
Comment (ansgarm) [inline examples/typescript/documentation/functions.ts:42]: What's this about? 😄

File: website/docs/cdktf/concepts/stacks.mdx [hunk 2]
```diff
@@ -690,63 +690,10 @@ If you want to keep the result of the function attached to one stack and save it
...
-  "source-stack-b",
-  "ami-2757f632"
-);
-
-new MyDependencyStack(dependencyApp, "dependency-stack", [
-  sourceStackA,
-  sourceStackB,
+this.allResources = new TerraformLocal(this, "merged_items", [
+  sourceStackA.instance.id,
+  sourceStackB.instance.id,
```
Comment (ansgarm) [inline website/docs/cdktf/concepts/stacks.mdx:695]: As described in #2579, I'd like to propose a change for this example. But it's fine for me to change this in the meantime.

**Review by ansgarm — APPROVED** [review]

File: examples/typescript/documentation/functions.ts (hunk 1 above)
Comment (mutahhir) [reply examples/typescript/documentation/functions.ts:42]: :D dunno, copied from original

Comment (ansgarm) [reply examples/typescript/documentation/functions.ts:42]: ah, okay :D

**Pull request merged**